_includes/head.html

```diff
@@ -1,14 +1,32 @@
+{% capture title %}{% if page.title %}{{ page.title }} | {% endif %}{{ site.title }} {% endcapture %}
+
 <head>
-	<title>{% if page.title %}{{ page.title }} | {% endif %}{{ site.title }}</title>
-	
+	<title>{{ title }}</title>
 	<!-- Meta -->
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<meta name="description" content="{{ site.description }}">
 	<meta name="author" content="HydPy Hyderabad">
 	<link rel="shortcut icon" href="favicon.ico">
-	
+	<meta name="robots" content="index, follow">
+
+	<meta name="title" content="{{ title }}">
+
+	<!-- Open Graph / Facebook -->
+	<meta property="og:type" content="website">
+	<meta property="og:url" content="https://pyconf.hydpy.org/2019/">
+	<meta property="og:title" content="{{ title }}">
+	<meta property="og:description" content="{{ site.description }}">
+	<meta property="og:image" content="assets/images/pyconflogo_ai_compressed.jpg">
+
+	<!-- Twitter -->
+	<meta property="twitter:card" content="summary_large_image">
+	<meta property="twitter:url" content="https://pyconf.hydpy.org/2019/">
```

Comment:
Should be the @pyconfhyd twitter url

Reply:
This seems to be correct Reference: https://davidwalsh.name/twitter-cards
I should add the twitter account details though in separate meta tag.

Reply:
Added `twitter:site` and `twitter:creator` details.

Reply:
Got it thanks. So this data will be used by twitter for preview